rotkehlchen/chain/ethereum/graph.py

```diff
@@ -56,7 +56,7 @@ def get_common_params(
     return param_types, param_values
 
 
-class Graph():
+class Graph:
```

Comment:
Thank god you removed all this empty parenthesis :pray:

Reply:
Just a new ruff rule. Consistency is great!